revista-latinoamericana-de-recursos-naturales.csl

```diff
@@ -0,0 +1,437 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="never" default-locale="en-GB">
+  <info>
+    <title>Revista Latinoamericana de Recursos Naturales</title>
+    <title-short>Rev. Lat. Rec. Nat.</title-short>
+    <id>http://www.zotero.org/styles/revista-latinoamericana-de-recursos-naturales</id>
+    <link href="http://www.zotero.org/styles/revista-latinoamericana-de-recursos-naturales" rel="self"/>
+    <link href="http://www.zotero.org/styles/harvard-author-date-es" rel="template"/>
```

Comment:
@rcervantesb, the "template" link, if used, must point to an existing style from our style repository. We don't have a "harvard-author-date-es.csl" style in https://github.com/citation-style-language/styles/.